bedrock/settings/base.py

```diff
@@ -126,8 +227,7 @@ def JINJA_CONFIG():
     path('media'),
 )
 
-JINGO_EXCLUDE_APPS = (
-    'admin', 'registration', 'rest_framework', 'rna', 'waffle')
+JINGO_EXCLUDE_APPS = ('admin', 'registration', 'rest_framework', 'rna', 'waffle')
```

Comment:
we've not stuck to 80 chars in months, so this seemed silly. I could make it a vertical list if you'd prefer.